### DISCUSSION TITLE
Are there any plans to support other types of artifacts besides OCI?

### DISCUSSION THREAD

**Discussion by hyeyoung-leee** [goharbor/harbor#18599] [Q&A] [answered]
Hi,

Are there any plans to support OS(rpm, apt) or language(java, python) package?
If there is no plan, I wonder if there is a way to provide various artifacts using harbor.

Thanks.

**Comment by Vad1mo** [comment, marked as answer]
No

**Comment by yemboado-ouoba** [comment]
Just a follow up on this thread. Could (in theory) Harbor be extended to support NuGet, Python, NPM, Maven packages? For example, if someone were to fork the code, could he add functionality to support these packages? if yes, how difficult would it be?
Thanks,

**Reply by Vad1mo** [reply]
Sure, but it should be done as an OCI proxy. It would be better if others supported OCI instead, this is IMO the general trend anyway. 
see https://www.bretfisher.com/cloud-native-devops-31/
